ex04.ex

```diff
@@ -75,7 +85,15 @@ defmodule Ex04 do
   return value will be the thing you have to manipulate.
   """
 
-  def even_odd . . . "your code"
+  def even_odd(list) do
```

Comment:
## -3

I have a dog that is trained to seek out people who write lines of code contain over 150 characters 🐕  Trust me, you don't want a visit.

In addition, there's a fair amount of duplication in there: both in terms of code and in the way it processes `list` twice.

How about this:

~~~ elixir
  def even_odd(list) do
    list
    |> reverse()
    |> reduce({ [], [] }, &partition_even_odd/2)
  end

  defp partition_even_odd(x, { es, os }) when Integer.is_even(x), do: {[x | es], os}
  defp partition_even_odd(x, { es, os }) when Integer.is_odd(x),  do: {es, [x | os]}
~~~